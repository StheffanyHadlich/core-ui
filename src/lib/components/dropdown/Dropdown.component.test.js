import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/dropdown/__snapshots__"
});
