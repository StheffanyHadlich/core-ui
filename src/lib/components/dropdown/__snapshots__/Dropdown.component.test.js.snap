// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Dropdown Default 1`] = `
<div
  className="storybook-dropdown sc-VigVT ciOuIf"
>
  <h3
    className="sc-jTzLTM eWkzeL"
  >
    Dropdown with text/icon
  </h3>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU cxyWUF"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="smaller"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="smaller"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        Help
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU boIWiS"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="small"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="small"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        Help
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU fOfyLU"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="base"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        Help
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU gyNVEX"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="large"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="large"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        Help
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU kWpnd"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="larger"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="larger"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        Help
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <h3
    className="sc-jTzLTM eWkzeL"
  >
    Dropdown with icon
  </h3>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU fOfyLU"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX bccMeP"
        size="base"
      >
        <i
          className="fas fa-star"
        />
      </span>
    </div>
  </div>
  <h3
    className="sc-jTzLTM eWkzeL"
  >
    Dropdown with text
  </h3>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU fOfyLU"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        Help
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <h3
    className="sc-jTzLTM eWkzeL"
  >
    Dropdown with variant
  </h3>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU fOfyLU"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="base"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        primary
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU hyupuE"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="base"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        secondary
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU jXvZUK"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="base"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        success
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU fGdmwK"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="base"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        info
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU fyffSG"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="base"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        warning
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
  <div
    className="sc-dropdown sc-htoDjs jLvNAM"
  >
    <div
      className="trigger sc-gqjmRU cRFtSq"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      size="base"
      tabIndex="0"
    >
      <span
        className="sc-ifAKCX joVDVz"
        size="base"
      >
        <i
          className="fas fa-star"
        />
      </span>
      <span
        className="sc-trigger-text sc-EHOje boinrg"
      >
        danger
      </span>
      <span
        className="sc-gZMcBi cfFiuj"
      >
        <i
          className="fas fa-caret-down"
        />
      </span>
    </div>
  </div>
</div>
`;
